src/test/java/com/mapzen/valhalla/InstructionTest.java

```diff
@@ -40,38 +41,68 @@
-            "8,\n" +
-            "0,\n" +
-            "\"1609m\",\n" +
-            "\"SE\",\n" +
-            "128\n" +
-            "]\n");
-
-    private static final JSONArray NON_INT_TURN_JSON = new JSONArray("[\n" +
-            "\"11-1\",\n" +
-            "\"19th Street\",\n" +
-            "1609,\n" +
-            "8,\n" +
-            "0,\n" +
-            "\"1609m\",\n" +
-            "\"SE\",\n" +
-            "128\n" +
-            "]\n");
-
-    private static final JSONArray STREET_NOT_FOUND = new JSONArray("[\n"
-            + "            \"11-3\",\n"
-            + "            \"{\\\"highway\\\":\\\"unclassified\\\", \\\"message\\\":\\\"name not found\\\"}\",\n"
-            + "            31,\n"
-            + "            5345,\n"
-            + "            5,\n"
-            + "            \"30m\",\n"
-            + "            \"N\",\n"
-            + "            8\n"
-            + "        ]");
+    private static final JSONArray JSON;
+
+    static {JSONArray JSON1;
+        try {
+            JSON1 = new JSONArray("[\n" +
+                    "\"10\",\n" +
+                    "\"19th Street\",\n" +
+                    "1609,\n" +
+                    "8,\n" +
+                    "0,\n" +
+                    "\"1609m\",\n" +
+                    "\"SE\",\n" +
+                    "128\n" +
+                    "]\n");
+        } catch (JSONException e) {
+            e.printStackTrace();
+            JSON1 = null;
+        }
+        JSON = JSON1;
+    }
+
+    private static final JSONArray NON_INT_TURN_JSON;
+
+    static {JSONArray NON_INT_TURN_JSON1;
+        try {
+            NON_INT_TURN_JSON1 = new JSONArray("[\n" +
+                    "\"11-1\",\n" +
+                    "\"19th Street\",\n" +
+                    "1609,\n" +
+                    "8,\n" +
+                    "0,\n" +
+                    "\"1609m\",\n" +
+                    "\"SE\",\n" +
+                    "128\n" +
+                    "]\n");
+        } catch (JSONException e) {
+            e.printStackTrace();
+            NON_INT_TURN_JSON1 = null;
+        }
+        NON_INT_TURN_JSON = NON_INT_TURN_JSON1;
+    }
+
+    private static final JSONArray STREET_NOT_FOUND;
+
+    static {JSONArray STREET_NOT_FOUND1;
+        try {
+            STREET_NOT_FOUND1 = new JSONArray("[\n"
+                    + "            \"11-3\",\n"
+                    + "            \"{\\\"highway\\\":\\\"unclassified\\\", \\\"message\\\":\\\"name not found\\\"}\",\n"
+                    + "            31,\n"
+                    + "            5345,\n"
+                    + "            5,\n"
+                    + "            \"30m\",\n"
+                    + "            \"N\",\n"
+                    + "            8\n"
+                    + "        ]");
+        } catch (JSONException e) {
+            e.printStackTrace();
+            STREET_NOT_FOUND1 = null;
+        }
+        STREET_NOT_FOUND = STREET_NOT_FOUND1;
+    }
```

Comment:
should we start thinking about adding these as fixture files? kinda annoying with all these escapes